src/android/aidl/types/primitives.md

```diff
@@ -0,0 +1,14 @@
+# Primitive Types
+
+Primitive types map (mostly) idiomatically:
+
+| AIDL Type | Rust Type | Note                                |
+| --------- | --------- | ----------------------------------- |
+| `boolean` | `bool`    |                                     |
+| `byte`    | `i8`      | Note that bytes are signed.         |
+| `char`    | `u16`     | Note the usage of `u16`, NOT `u32`. |
```

Comment:
Depending which language you come from, you'd expect a different type here: `char` would be `u8` coming from C, `u16` from Java, or `u32` from Rust. AIDL picked the Java option.